src/follower.py

```diff
@@ -186,9 +187,10 @@ def follow(self):
 def main():
     """Instantiates a Follower object and continously calls Follower.follow()."""
     follower = Follower()
+    log = Log()
```

Comment:
I didnt fix the follower, I think we talked about it last night and you said to wait until you finished making changes to the follower otherwise I would get merge conflicts.

Reply:
This has absolutely nothing to do with the follower. This error is completely on your end. What happens if you create a Log object with no parameter?

Reply:
The correct version is in the leader where log = Log('follower') is given a parameter. 
This is in the follower file and has not been updated to match the leader and the log. Would you like me to wait or to change it to match right now?

Reply:
When I said don't make changes to the error we have in the follower, I meant not to change opencv_to_wheels.